scripts/poppunk_iterate.py

```diff
@@ -260,17 +260,22 @@ def is_nested(cluster_dict, child_members, node_list):
     for leaf in tree.traverse_leaves():
         parent_node = leaf.get_parent()
         if not parent_node.is_root():
+            if parent_node.get_edge_length() < args.cutoff and parent_node.get_parent().is_root():
+                cut_clusters.add(parent_node)
```

Comment:
if the grand-parent node is the root node and parent node.dist < cutoff, we keep this parent_node